### PR TITLE
Feat: Display funding in trade history table and update layout

### LINKED ISSUE #2538: Feat: Display accrued funding in trade history
### Description

Add a column in trade history for accrued funding.

This data should already be available from the subgraph trade history.

### PR DESCRIPTION
**Pull request by avclarke**
## Description
1. Add a column for accrued funding in the trade history table
2. Update the table layout (desktop/tablet)

## Related issue
closes #2538

## Motivation and Context
Popular request from traders

### Desktop / Dark
截屏2023-07-21 13 28 59

### Desktop / Light
截屏2023-07-21 13 29 50

### Tablet / Dark
截屏2023-07-21 13 38 05

### Tablet / Light
截屏2023-07-21 13 38 22

### CODE REVIEW

File: packages/app/src/sections/futures/PositionType.tsx [hunk 1]
```diff
@@ -6,23 +6,30 @@ import Body from 'components/Text/Body'
...
-		<StyledText side={side}>{side}</StyledText>
+		<StyledText side={side} variant={variant}>
+			{side}
+		</StyledText>
 	)
 }
 
 const StyledText = styled(Body).attrs({ weight: 'bold', capitalized: true, type: 'span' })<{
 	side: PositionSide
+	variant: 'trades' | 'default'
```
Comment (avclarke) [inline packages/app/src/sections/futures/PositionType.tsx:28]:
```suggestion
	variant: 'badge' | 'text'
```

This way the style is described for other components to use

File: packages/app/src/sections/futures/Trades/Trades.tsx [hunk 2]
```diff
@@ -67,80 +71,220 @@ const Trades = memo(() => {
...
 					),
-					accessorKey: 'amount',
 					sortingFn: 'basic',
-					cell: (cellProps) => (
-						<>{formatCryptoCurrency(cellProps.getValue(), { suggestDecimals: true })}</>
+					accessorKey: 'feesPaid',
+					cell: (cellProps) => {
+						return (
+							<FlexDivCol>
+								{cellProps.getValue().eq(0) ? (
```
Comment (avclarke) [inline packages/app/src/sections/futures/Trades/Trades.tsx:180]: Not sure I understand this conditional, what are the differences in display when the value is 0?

File: packages/app/src/sections/futures/PositionType.tsx (hunk 1 above)
Comment (LeifuChen) [reply packages/app/src/sections/futures/PositionType.tsx:28]: Yes. It's more understandable. Updated.

File: packages/app/src/sections/futures/Trades/Trades.tsx (hunk 2 above)
Comment (LeifuChen) [reply packages/app/src/sections/futures/Trades/Trades.tsx:180]: That's true. Let's keep it simple. Updated.

**Review by LeifuChen — APPROVED** [review]

**Pull request merged**